apps/veil/package.json

```diff
@@ -97,5 +99,8 @@
     "typescript": "5.5.3",
     "typescript-eslint": "^8.17.0",
     "vitest": "^1.6.0"
+  },
+  "engines": {
```

Comment:
fyi: this field in Veil was causing GitHub actions to fail, so we removed it recently and did this: https://github.com/penumbra-zone/web/pull/2204

Reply:
Great, it was causing painful issues with pnpm for me as well. This is a rebase artifact.